### ISSUE TITLE
Possibility to add UN to the list of providers

### ISSUE THREAD

**Issue by Tungurahua** [amattioc/SDMX#51]
Is it possible to add UNdata to the list of providers? They seem to run a SDMX based REST service though I don't know about it's maturity and if it runs SDMX2.1

**Comment by amattioc** [comment]
Hi, do you have additional information about the provider?

I need to find the REST URL...

**Comment by Tungurahua** [comment]
Sorry, somehow the links slipped my prior post. The information I found about the interface is here:
http://data.un.org/Host.aspx?Content=API

However, it's version v0.14.6 Beta so I guess it's not yet fully functional.  With some further reading I found that the current implementation is SOAP and they are close to rolling out a REST service. So it's unlikely that a public REST URL already exists.

So right now there is probably no possibility to access UNstat via REST, but as soon as I hear of any updates, I will post them here.

Best
Albrecht
